packages/ui/src/components/modals/ChangeMultisig.tsx

```diff
@@ -356,9 +365,17 @@ const ChangeMultisig = ({ onClose, className }: Props) => {
       fullWidth
       maxWidth={'sm'}
       open
-      onClose={handleClose}
       className={className}
     >
+      <IconButton
+        className="closeButton"
+        size="small"
+        aria-label="close"
+        color="inherit"
+        onClick={onClose}
+      >
+        <CloseIcon fontSize="small" />
+      </IconButton>
```

Comment:
Add the check on `isCallStep` to keep the previous behavior
```suggestion
      {!isCallStep && (
        <IconButton
          className="closeButton"
          size="small"
          aria-label="close"
          color="inherit"
          onClick={onClose}
        >
          <CloseIcon fontSize="small" />
        </IconButton>
      )}
```